test/c/CMakeLists.txt

```
@@ -19,12 +19,14 @@ file (
 # test file.
 create_test_sourcelist (source_files test_driver.c ${discovered_tests})
 
+add_library(common common.c)
+install(TARGETS common RUNTIME DESTINATION ${CMAKE_BINARY_DIR})
```

With a static lib we shouldn't need to worry about placing the DLLs in the right location on windows
```suggestion
add_library(common STATIC common.c)
```